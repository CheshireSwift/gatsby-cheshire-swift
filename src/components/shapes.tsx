import * as React from 'react';

export const Nought = () => (
  <svg version="1.1" baseProfile="full" width="100%" height="100%">
    <circle
      cx="50%"
      cy="50%"
      r="30%"
      stroke="black"
      strokeWidth="10%"
      fill="transparent"
    />
  </svg>
);

export const Cross = () => (
  <svg version="1.1" baseProfile="full" width="100%" height="100%">
    <line
      x1="20%"
      y1="20%"
      x2="80%"
      y2="80%"
      stroke="black"
      strokeWidth="10%"
    />
    <line
      x1="20%"
      y1="80%"
      x2="80%"
      y2="20%"
      stroke="black"
      strokeWidth="10%"
    />
  </svg>
);
