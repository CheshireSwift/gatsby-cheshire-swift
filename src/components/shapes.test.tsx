import * as React from 'react';
import { render } from 'enzyme';

import { Nought, Cross } from './shapes';

describe('Nought', () => {
  it('matches the snaphot', () => {
    expect(render(<Nought />)).toMatchSnapshot();
  });
});

describe('cross', () => {
  it('Cross matches the snaphot', () => {
    expect(render(<Cross />)).toMatchSnapshot();
  });
});
