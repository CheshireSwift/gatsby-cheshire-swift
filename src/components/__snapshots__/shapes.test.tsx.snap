// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nought matches the snaphot 1`] = `
<svg
  baseProfile="full"
  height="100%"
  version="1.1"
  width="100%"
>
  <circle
    cx="50%"
    cy="50%"
    fill="transparent"
    r="30%"
    stroke="black"
    stroke-width="10%"
  />
</svg>
`;

exports[`cross Cross matches the snaphot 1`] = `
<svg
  baseProfile="full"
  height="100%"
  version="1.1"
  width="100%"
>
  <line
    stroke="black"
    stroke-width="10%"
    x1="20%"
    x2="80%"
    y1="20%"
    y2="80%"
  />
  <line
    stroke="black"
    stroke-width="10%"
    x1="20%"
    x2="80%"
    y1="80%"
    y2="20%"
  />
</svg>
`;
